### ISSUE TITLE
[Channel dev, v0.3.6] I can build flutter_gallery, but the boot will crash after installation.

### ISSUE THREAD

**Issue by dengsgo** [flutter/flutter#17334]
I can build flutter_gallery ( use `flutter build apk` ), but the boot will crash after installation.

![_20180506133523](https://user-images.githubusercontent.com/7929002/39670221-7883dc28-5132-11e8-92fc-a6452f88aa57.jpg)

![_20180506133534](https://user-images.githubusercontent.com/7929002/39670223-8087029c-5132-11e8-9635-0b8fde08a010.jpg)

 running `flutter doctor -v`：

```
PS D:\flutter\examples\flutter_gallery> flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel dev, v0.3.6, on Microsoft Windows [Version 10.0.17134.1], locale zh-CN)
[!] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    X Android license status unknown.
[X] Android Studio (not installed)
[√] IntelliJ IDEA Community Edition (version 2018.1)
[!] Connected devices
    ! No devices available

! Doctor found issues in 3 categories.
PS D:\flutter\examples\flutter_gallery>
```

**Comment by fly512** [comment]
V0.3.6 is not available, using v0.3.5

**Comment by dengsgo** [comment]
> V0.3.6 is not available, using v0.3.5 

@fly512  I upgraded the version with the `flutter upgrade` command, and I don't know why it was upgraded to an unusable version.
Anyway, thank you for your answer
